docs/generated/go.md

```diff
@@ -0,0 +1,201 @@
+---
+bodyclass: docs
+headline: Go Generated Code Guide
+layout: docs
+sidenav: doc-side-quickstart-nav.html
+type: markdown
+---
+
+<p class="lead">This guide gives an overview of code generated with the protoc grpc go plugin.</p>
+
+<div id="toc"></div>
+
+## Compiler Invocation and Results of Code Generation
+This section provides instructions for using the `grpc` plugin to `protoc-gen-go`, when compiling `.proto` files with `protoc`.
```

Comment:
This section provides instructions for using the `grpc` plugin when generating code from .proto files with `protoc` and `protoc-gen-go`.

(just generally, this sounds like grpc for Go is a plugin to protoc-gen-go, which is a plugin for protoc - can a plugin have a plugin?)